crd/src/util.rs

```diff
@@ -174,10 +174,28 @@ pub async fn get_opa_connection_info(
     Ok(OpaConnectionInformation { connection_string })
 }
 
+/// Remove redundant "/" from the url path
```

Comment:
What does "redundant" mean here?
I don't understand the code to be honest.

Reply:
Thats for the created opa url. I hardcoded the slashes in the format!(...) and now you can e.g. refer to the rule either like:
/package/ or package/ etc.

Reply:
Okay, could you document this?

Something like:
Removes slashes ("/") from the beginning or end of a string

But...I don't really understand the code so it's up to you to describe what the function actually does :)